Library/Tests/ViewModels/LiveStreamContainerViewModelTests.swift

```diff
@@ -831,4 +829,94 @@ internal final class LiveStreamContainerViewModelTests: TestCase {
 
     self.goToProjectButtonContainerHidden.assertValues([false])
   }
+
```

Comment:
👇 These tests just run through a bunch of scenarios in which we want to be sure our `LiveStreamViewController` will still be configured.